src/main/java/de/unistuttgart/crosswordbackend/crosswordchecker/CrosswordChecker.java

```diff
@@ -0,0 +1,22 @@
+package de.unistuttgart.crosswordbackend.crosswordchecker;
+
+import de.unistuttgart.crosswordbackend.data.Crossword;
+import de.unistuttgart.crosswordbackend.data.Question;
+import java.util.List;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+public class CrosswordChecker {
+
+  /**
+   * creates a crossword with the given questions and return if every answer is placed
+   * on the crossword-puzzle
+   * @param questions questions with answers for a crossword-puzzle
+   * @return if all answers are placed on the crossword-puzzle return true else return false
+   */
+  public boolean checkCrossword(List<Question> questions) {
+    CrosswordGenerator crosswordGenerator = new CrosswordGenerator();
+    Crossword crossword = crosswordGenerator.generateCrossword(questions);
+    return crossword.getAnswer().isEmpty();
```

Comment:
```suggestion
    return new CrosswordGenerator().generateCrossword(questions).getAnswer().isEmpty();
```

Reply:
What I also noticed:
While it is possible to do this using booleans, the normal approach would be to throw a specific exception, i.e. `CrosswordGenerationException` when the crossword can't be generated.

Reply:
This algorithm is a close copy of the algorithm used in the frontend. In the frontend, if it is not possible to create a crossword with all the questions, the remaining questions get deleted and you get a crossword with fewer questions than you put in.

Reply:
**Now** I have objections.
Never a good idea to decide for the user what he wants (without notifying him that this was done).